nb2/05.08-V2-Random-Forests.ipynb

```diff
@@ -0,0 +1,728 @@
+{
```

Comment:
This is comment 1
<br/>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>